Makefile

```diff
@@ -0,0 +1,184 @@
+##@ Build openstack-ansibleee-runner image
```

Comment:
Before the first target, add some useful variables that we lost in the last edit:
```

# openstack-ansibleee-runner image
EEIMG ?= quay.io/openstack-k8s-operators/openstack-ansibleee-runner

# edpm-ansible local repo
EDPM_LOCAL_REPO ?= ""

VERIFY_TLS ?= true


```